docs/next/util/navigation.ts

```diff
@@ -49,10 +50,16 @@ export const latestAllPaths = () => {
     });
 };
 
-export const latestAllDynamicPaths = () => {
+export const latestAllDynamicPaths = (excludeNonMdx = false) => {
```

Comment:
Boolean function params are confusing -- consider making the param a config object, e.g. `{excludeNonMdx: boolean}`?